src/transformers/tokenization_utils_base.py

```diff
@@ -1781,25 +1781,22 @@ def _from_pretrained(
             if config_tokenizer_class is None:
                 # Third attempt. If we have not yet found the original type of the tokenizer,
                 # we are loading we see if we can infer it from the type of the configuration file
-                from .models.auto.configuration_auto import CONFIG_MAPPING  # tests_ignore
-                from .models.auto.tokenization_auto import TOKENIZER_MAPPING  # tests_ignore
+                from .models.auto.tokenization_auto import TOKENIZER_MAPPING_NAMES  # tests_ignore
```

Comment:
Changes here are to use the name mappings (since we only want the names of the class), which go model_type to tokenizer classes.